openedx/openedx-platform
# [BB-2961] [BD-05] [TNL-7577] Added open response button to the component tray

The author needs to have quick access to the ORA menu in order
to get proceed with the assessment design. This introduces the ORA
button to the tray.

**JIRA tickets**: [TNL-7577](https://openedx.atlassian.net/browse/TNL-7577)

**Discussions**: None

**Dependencies**:  This is a dependency on presets depending on how to implement 
https://github.com/edx/edx-ora2/pull/1502

**Screenshots**: 

![image](https://user-images.githubusercontent.com/7670449/93531161-641ab200-f95c-11ea-9cf8-9fb5e301d899.png)


**Sandbox URL**: [Sandbox](https://studio.pr25012.sandbox.opencraft.hosting/
)
**Merge deadline**: "None" 

**Testing instructions**:

1. It requires the master devstacks
2. Pull this branch in edx-platform
3. We need to run `studio` and `lms`
4. Run `make dev.up.lms+studio`
5.  Login to studio with `edx account`
6. Go to the demo course edit a unit and try adding exercises
7. You will see the `Open Response` button ( just like in the screenshot) clicking that will have the Assessment template populated
8. if you click on Problem button, it shouldn't show `Open Assessment Response`. You can check this by clicking on `Problem` --> `Advance` tab, there shouldn't be ORA  present there.
9. To run automated tests you can run `paver test_system -t cms/djangoapps/contentstore/views/tests/test_item.py::TestComponentTemplates --fasttest`


**Reviewers**
